File: Beecrowd/1114 Senha Fixa/Main.java
import java.util.Scanner;

public class Main {
    public static void main(String[] args) throws Exception {
        Scanner sc = new Scanner(System.in);
        int senha;

        while(true){
            senha = sc.nextInt();

            if(senha == 2002){
                System.out.println("Acesso Permitido");
                break;
            }
            else{
                System.out.println("Senha Invalida");
            }
        }

        sc.close();
    }
}

File: Beecrowd/1114 Senha Fixa/main.cpp
#include <iostream>
 
using namespace std;
 
int main() {
 
	int senha;
	
	do{
		cin >> senha;
	
	if(senha == 2002)
		cout <<	"Acesso Permitido" << endl;
	else
		cout << "Senha Invalida" << endl;
	}while(senha != 2002);
 
    return 0;
}